vite.config.ts

```diff
@@ -2,13 +2,29 @@ import { defineConfig } from 'vite'
 import react from '@vitejs/plugin-react-swc'
 import svgr from 'vite-plugin-svgr'
 import path from 'path'
+import htmlConfig from 'vite-plugin-html-config'
+
+import htmlConfigOptions from './config/html-config'
 
 // https://vitejs.dev/config/
 export default defineConfig({
-  plugins: [react(), svgr()],
+  plugins: [
+    htmlConfig(htmlConfigOptions),
+    react(), 
+    svgr()
+  ],
   resolve: {
     alias: {
       '@': path.resolve(__dirname, './src')
     }
-  }
+  },
+  build: {
+      rollupOptions: {
+        output: {
+          manualChunks: {
+            'react-vendor': ['react', 'react-dom']
```

Comment:
Maybe we could also add `react-router-dom` to this vendor, no?

Reply:
Ah ah, yes good catch. When I started the PR there was no router yet that's why. I add it of course.